src/store/authors/reducer.js

```diff
@@ -0,0 +1,15 @@
+import * as actions from './types.js';
+
+export const authorsInitialState = [];
+
+export const authorsReducer = (state = authorsInitialState, action) => {
+	switch (action.type) {
+		case actions.SAVE_AUTHORS:
```

Comment:
```suggestion
		case actions.SAVE_AUTHOR:
```
you are adding one at a time, so singular